airflow/providers/google/marketing_platform/example_dags/example_display_video.py

```diff
@@ -56,6 +57,13 @@
 PARAMS = {"dataRange": "LAST_14_DAYS", "timezoneCode": "America/New_York"}
 # [END howto_display_video_env_variables]
 
+# download_line_items variables
+REQUEST_BODY = {
+    "filterType": ADVERTISER_ID,
+    "format": "CSV",
+    "fileSpec": "EWF"
+}
+
```

Comment:
Can you add this part of example to how to guide?

Reply:
Yes, sure.

Reply:
@michalslowikowski00 should we use `START` and `END` markers to avoid code duplication here and in docs?